src/compiler/docs/test/generate-doc-data.spec.ts

```diff
@@ -283,12 +290,13 @@ auto-generated content
 
   describe('default values', () => {
     it.each(['', null, undefined])(
-      'defaults the annotation to an empty string if %s is provided',
+      "defaults the annotation to an empty string if '%s' is provided",
```

Comment:
I noticed this output was previously reading as:

> defaults the annotation to an empty string if is provided

for the case of the empty string in one of my new tests, and went back and fixed it in a few places here. I can pull it out if folks wish, but it was only two or so tests affected - didn't seem worth it. LMK if you think otherwise, happy to pull it out

Reply:
it's all good 👍